clash_lib/src/session.rs

```diff
@@ -12,6 +12,13 @@ use tokio::io::{AsyncRead, AsyncReadExt};
 
 use erased_serde::Serialize as ESerialize;
 
+// mark of the packet from clash-rs
+static DEFAULT_PACKET_MARK: u32 = 0xff;
+
+pub(crate) fn get_packet_mark() -> u32 {
+    DEFAULT_PACKET_MARK
+}
+
```

Comment:
What i thought is:
```rust
static GLOBAL_PACKET_MARK: AtomicU32 = AtomicU32::new(0);

pub(crate) fn get_packet_mark() -> Option<u32> {
    let mark = GLOBAL_PACKET_MARK.load(Ordering::Relex)
    if mark != 0 {
        return Some(mark)
    } else {
        return None
    }
}
```

if tun auto-route or tproxy auto-route is disabled, we can just set this to 0.
Then when we constructing Session, we can use it(`get_packet_mark`).